src/app/util/attribute-metadata.h

```diff
@@ -105,6 +105,17 @@ union EmberAfDefaultOrMinMaxAttributeValue
     const EmberAfAttributeMinMaxValue * ptrToMinMaxValue;
 };
 
+enum class EmberAfAttributeWritePermission
+{
+    DenyWrite            = 0,
+    AllowWriteNormal     = 1,
+    AllowWriteOfReadOnly = 2,
+    UnsupportedAttribute = 134,
+    InvalidValue         = 135,
+    ReadOnly             = 136,
+    InvalidDataType      = 141,
```

Comment:
Maybe add a comment about how these need to match spec status values?

Reply:
I have added a comment with the matching InteractionModel status equivalent next to each of these.